highlevel/robot/entity/configuration.py

```diff
@@ -26,17 +28,44 @@ class Configuration:
     """
     Hold configuration parameters.
     """
+    debug: DebugConfiguration
+
+    # Strategy config
     initial_position: Vector2
     initial_angle: Radian
     color: Color
+    field_shape: Tuple[Millimeter, Millimeter]
 
+    # Motion control:
+    # # Robot data
     robot_width: Millimeter
     robot_length: Millimeter
-
-    field_shape: Tuple[Millimeter, Millimeter]
-
     wheel_radius: Millimeter
     encoder_ticks_per_revolution: int
     distance_between_wheels: Millimeter
+    encoder_update_rate: int
```

Comment:
Use proper unit here please i.e. `Hz

small descriptions would also be appreciated